server/src/transactions/entities/transaction.entity.ts

```diff
@@ -0,0 +1,40 @@
+import { Entity, PrimaryGeneratedColumn, Column, ManyToOne, JoinColumn } from 'typeorm';
+import { TransactionStatus } from '../transaction-status.enum';
+import { User } from '../../users/entities/user.entity';
+import { Asset } from '../../assets/entities/asset.entity';
+import { Organization } from '../../organizations/entities/organization.entity';
+
+@Entity('transactions')
+export class Transaction {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({
+    type: 'enum',
+    enum: TransactionStatus,
+    default: TransactionStatus.IN_PROGRESS,
+  })
+  status: TransactionStatus;
+
+  @Column({
+    type: 'date',
+    default: new Date().toLocaleString(),
+  })
+  created_date: Date;
```

Comment:
### Should fix:
Here, you are using Javascript dates, but we should let TypeORM use their SQL timestamp decorator:

https://typeorm.io/#/decorator-reference/createdatecolumn

```typescript
@Entity()
export class User {

    @CreateDateColumn()
    createdDate: Date;

}
```